apiary.apib

```diff
@@ -766,9 +766,86 @@ This operations is available only for organization administrators.
     + Body
 
             {
+                "uri": "https://connection.keboola.com/v2/storage/buckets/in.c-my-bucket",
+                "id": "in.c-my-bucket",
+                "name": "c-my-bucket",
+                "stage": "in",
+                "description": "Some Description",
+                "tables": "https://connection.keboola.com/v2/storage/buckets/in.c-my-bucket/tables",
+                "created": "2017-02-13T12:01:05+0100",
+                "lastChangeDate": null,
+                "isReadOnly": false,
+                "dataSizeBytes": 0,
+                "rowsCount": 0,
+                "isMaintenance": false,
+                "backend": "snowflake",
+                "sharing": "organization"
+            }
+
+### Share Bucket To Organization [POST /v2/storage/buckets/{bucket_id}/share-organization]
+[Enables sharing](https://help.keboola.com/storage/buckets/sharing/) or **update sharing** of a bucket.
+Bucket is shared to organization. Any organization member can link bucket to project in organization.
+This operations is available only for organization administrators.
++ Parameters
+    + bucket_id (required) - Bucket Id
+
++ Request
+    + Headers
+
+            X-StorageApi-Token: your_token
+
++ Response 202 (application/json)
+    + Body
+
+            {
+                "uri": "https://connection.keboola.com/v2/storage/buckets/in.c-my-bucket",
+                "id": "in.c-my-bucket",
+                "name": "c-my-bucket",
+                "stage": "in",
+                "description": "Some Description",
+                "tables": "https://connection.keboola.com/v2/storage/buckets/in.c-my-bucket/tables",
+                "created": "2017-02-13T12:01:05+0100",
+                "lastChangeDate": null,
+                "isReadOnly": false,
+                "dataSizeBytes": 0,
+                "rowsCount": 0,
+                "isMaintenance": false,
+                "backend": "snowflake",
                 "sharing": "organization"
             }
 
+### Share Bucket To Organization Project [POST /v2/storage/buckets/{bucket_id}/share-organization-project]
+[Enables sharing](https://help.keboola.com/storage/buckets/sharing/) or **update sharing** of a bucket.
+Bucket is shared to organization. Any member of the organization or member of a project in the organization can link bucket to project.
+This operations is available only for organization administrators.
```

Comment:
Ok, I think this is better again, sorry:
`Enable sharing for this bucket within the organization. Any member of a project within the organization will be able to link to this bucket from their project. 
This operations is available only for organization administrators.`